tasks/libs/pipeline_notifications.py

```diff
@@ -39,6 +39,20 @@ def read_owners(owners_file):
         return CodeOwners(f.read())
 
 
+def check_owners_slack(print_missing_teams=True, owners_file=".github/CODEOWNERS"):
```

Comment:
```suggestion
def check_for_missing_owners_slack(print_missing_teams=True, owners_file=".github/CODEOWNERS"):
```

Because if the function returns True for `check_owners_slack` it feels like everything went well, so this makes more sense IMO, WDYT?